cmd/testoutput/main.go

```diff
@@ -33,21 +33,35 @@ func main() {
 	fmt.Println(string(bts))
 }
 
-func readFile(file string) ([][]byte, error) {
-	bts, err := ioutil.ReadFile(file)
+func readStream() ([][]byte, error) {
+	_, err := os.Stdin.Stat()
 	if err != nil {
-		return nil, err
+		panic(err)
```

Comment:
better return the error here and let the `main()` function do the `log.Panic(err)`.

Reply:
I didn't notice that's what it was doing, but that makes a lot more sense!